### PR TITLE
Ops have an intelligence potion instead of PDA bombs

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
del: Nuclear operatives no longer are issued a free PDA with a
Detomatix cartridge.
add: Instead, they are given a syndicate intelligence potion, which may
be used on any simple animal in the vicinity.
/:cl:

image

PDA bombs are a new player trap. Everyone at this point associates PDA
bombs with nuke ops, because no one really uses them as a regular
traitor since they're not cost effective.

With this PR, ops (most of the time) gain a talking unlimited
jetpack that can be injected with explosives. Detomatix catridges can
still be bought.

Although the ops are losing 6 TC of equipment, and gaining only 4 TC, I
think this is far more suited to their aims and plans.

### CODE REVIEW

**Comment by nicbn** [comment]
If no one observes the game the potion will be useless until someone dies

**Comment by Borisvanmemes** [comment]
@nicbn Just go warops then, loads of people suicide when they see warops.

**Comment by PKPenguin321** [comment]
:-1:
- PDA bombs being a war ops "tell" isn't inherently a bad thing (if it's even true), since it means regular traitors can trick the crew into thinking there are ops
- Having extra people on your team is VASTLY superior to PDA bombs, even if that extra person is a space carp. Ops don't need it by default

**Comment by Tlaltecuhtli** [comment]
didnt they have the sentience potion already?

**Comment by subject217** [comment]
>Everyone at this point associates PDA
bombs with nuke ops, because no one really uses them as a regular
traitor since they're not cost effective.

I disagree, I've encountered lots of Detomatix cartridges from traitors. Also the PDA itself is quite handy and has a couple of niche functionalities.

**Comment by Bawhoppen** [comment]
This is silly and doesn't make sense from an in-game perspective.

**Comment by coiax** [comment]
How does it not make sense from an ingame perspective?

**Comment by Arkvold** [comment]
> As traitor with detomatix cartridge, see security officer inside checkpoint station through window
> Need to get past checkpoint, not robust enough to break in and take out officer
> Not really sneaky or charming enough to fool him either
> Blow his PDA.  Likely destroy ID that was inside.
> Now he can't leave the checkpoint to stop me without smashing a window or tearing down a wall, since he can't open sec doors now.
>If he didn't keep his ID inside the PDA, he's probably freaking out and raising alarm about possible nuke ops, might be able to slip in during chaos (everyone's expecting a squad of red-hardsuited stormtroopers, not a lone greyshirt with an agenda.)
>At a minimum, he needs a new PDA, and only the HoP can get him one, necessitating a delivery (ambush the delivery boy, steal his clothes and ID) or the officer leaving his station.

This is just one way det carts can be super-useful to any traitor, not just nuke ops.  Sentience potion makes far less sense than a cartridge that lets you deny the enemy a communication method and possibly eliminate their access to their own station if they keep their ID in their PDA.

While I normally support anything that would allow more players in the round, sentience potion is pretty nonsensical, early-round options for using it are limited, and there's really only two options for using it: carp plushie (requiring additional TC to get) or (once you've reached the HoP's office) Ian.

A syndi sentience pot would also be a dead giveaway that you're ops - even more than a det cart would be.  Nobody outside of xenobio has sentient carp helping them out, so if there's carp outside the station helping people towards airlocks, and no carp migration warning, it's a dead giveaway that there's nuke ops present.

**Comment by coiax** [comment]
Maybe I should just straight up give ops a free sentience potion, since people seem very passionate about the Detomatix PDA.

**Comment by Arkvold** [comment]
Why the sentience potion, anyway?

I mean, for a nuke ops team, the sentience potion is only really useful for player-controlled carp allies.  While that shouldn't be underestimated, and it does allow more player-controlled entities in the round, there's not a lot space carp can do aside from swim around and gnaw things.  Honestly, once the novelty of being a sentient simplemob wears off, it becomes pretty boring from a player perspective, and there's not a lot of opportunity for them to get into a role that's more engaging.

Actually _using_ the sentience potion requires buying an additional 1TC item unless you wait until you're on the station.  If you decide to wait, then you also have to consider that from a combat perspective, pets like Ian are a lot less formidable than carp.  Aside from spying (which is difficult to do as a sentient simplemob pet since people _expect_ them to remain at their owner's post, if for no other reason than to keep the chef or clown away from them), sentient pets are a liability for a nuke ops team, and using carp is basically going loud from the word 'go.'

Unless you carefully wait for Xenobio to work with gold slime reactions, you're not going to grant sentience to any viable combat creatures other than carp, and if you have access to them, so does the station crew.  If you can wait and stay infiltrated _that_ long, you probably could've already won the round by more direct means.  In the end, sentience potions really don't have a lot of usefulness for quiet nuke ops jobs - people will notice if a sentient pet isn't behaving like a simplemob, especially if said pet emotes or speaks, like Ian or Poly, and then they're as good as dead unless they're combat-capable.

Finally, I question why the _sentience potion_ is necessary, specifically.  Nuke-ops teams are already one of the strongest roundstart fighting forces in SS13, ranking only just behind ERT troopers and Death Squad.  Do they really need to be able to double their numbers by creating spaceworthy ~~ablative meatshields~~ attack-fish for 1 TC/member?  Because that's what you're adding to nuke ops, if you gave every one of them a free sentience potion, whether it's at the cost of the det pda or not.

TL;DR - Sentience potion instead of det pda is enormously powerful for loud, direct strategies, but is borderline useless for infiltration techniques.  Regardless of how it's used, the role of a sentient simplemob is going to have a narrow profile of activities to participate in, and it's probably going to be really short once you either charge into combat as a hungry space-carp, or get outed as a traitorpet and butchered by the chef.
